en/docs/learn/examples/endpoint-examples/using-static-recepient-list-endpoints.md

```diff
@@ -1,69 +1,82 @@
+                <payloadFactory media-type="json" template-type="default">
+                    <format>{"payload": "sample"}</format>
+                    <args></args>
+                </payloadFactory>
+                <sequence key="EndpointCaller"/>
                 <respond/>
             </inSequence>
             <faultSequence>
                 <sequence key="ErrorHandler"/>
             </faultSequence>
-        </target>
-    </proxy>
+        </resource>
+    </api>
+    ```
+
+=== "Recipient List Endpoint"
+    ```xml
+    <?xml version="1.0" encoding="UTF-8"?>
+    <endpoint name="RecipientListEndpoint" xmlns="http://ws.apache.org/ns/synapse">
+        <recipientlist>
+            <endpoint>
+                <http method="post" uri-template="<URI_TEMPLATE_01>"/>
+            </endpoint>
+            <endpoint>
+                <http method="post" uri-template="<URI_TEMPLATE_02>"/>
+            </endpoint>
+            <endpoint>
+                <http method="post" uri-template="<URI_TEMPLATE_03>"/>
+            </endpoint>
+        </recipientlist>
+    </endpoint>
+    ```
+
+=== "Endpoint Calling Sequence"
+    ```xml
+    <?xml version="1.0" encoding="UTF-8"?>
+    <sequence name="EndpointCaller" trace="disable" xmlns="http://ws.apache.org/ns/synapse">
+        <call>
+            <endpoint key="RecipientListEndpoint"/>
+        </call>
+        <aggregate>
+            <onComplete expression="json-eval($)">
+                <log level="full"/>
+            </onComplete>
+        </aggregate>
+    </sequence>
     ```
+
 === "Error Handling Sequence"    
     ```xml
     <sequence name="ErrorHandler" xmlns="http://ws.apache.org/ns/synapse">
         <makefault response="true" version="soap12">
             <code xmlns:soap12Env="http://www.w3.org/2003/05/soap-envelope" value="soap12Env:Receiver"/>
-            <reason value="COULDN'T SEND THE MESSAGE TO THE SERVER."/>
+            <reason value="Couldn't send the request to the server"/>
         </makefault>
         <respond/>
     </sequence>
     ```
 
-<!--
-Set up the back-end service.
-
-Invoke the Micro Integrator:
-
-To test this, run
-the StockQuote client to send an out-only message as follows:
+## Testing the scenario
 
-```bash
-ant stockquote -Dmode=placeorder -Dtrpurl=http://localhost:8280/
-```
-
-If you examine the console output of
-each server, you can see that requests are processed by the three
-servers as follows:
-
-```bash
-Accepted order #1 for : 15738 stocks of IBM at $ 185.51155223506518
-```
+Open a terminal and execute the following curl to invoke the API.
+    ```bash
+    curl --location http://localhost:8290/callEndpoint/
+    ```
 
-Now shutdown MyServer1 and resend the request. You will observe that requests are still processed by MyServer2 and MyServer3.
--->
+The response received will be the aggregated output of the responses received from each endpoint which shows that the payload was cloned and it was sent to all the endpoints.
```

Comment:
Let's add this as a note and lets say this is done as the aggregate mediator is used.

Reply:
Fixed in 3fe4427f996a276d6b2b23d3b3ac3351b37cda10